examples/interpret.rs

```diff
@@ -12,15 +14,32 @@ fn main() {
         return;
     }
 
+    // Intrepreter initialization.
+    // parity_wasm::ProgramInstance can be parameterized with a custom User error to be returned from native modules
+    // parity_wasm::DefaultProgramInstance parametrize ProgramInstance with a pre-defined "DummyUserError"
```

Comment:
No reason to mention anything about user errors here, since it will be changed soon and also hardly related to the example